frontend/src/component/photo/clipboard.vue

```diff
@@ -209,15 +210,25 @@
                 this.clearClipboard();
             },
             download() {
-                if (this.selection.length === 1) {
-                    this.onDownload(`/api/v1/photos/${this.selection[0]}/dl?t=${this.$config.downloadToken()}`);
-                } else {
-                    Api.post("zip", {"photos": this.selection}).then(r => {
-                        this.onDownload(`/api/v1/zip/${r.data.filename}?t=${this.$config.downloadToken()}`);
-                    });
-                }
-
-                this.expanded = false;
+              let photoUIDs = [];
+              for (let i = 0; i < this.selection.length; i++) {
+                new Photo().find(this.selection[i]).then(photo => {
+                  for (let j = 0; j < photo.Files.length; j++) {
+                    window.console.log(photo.Files);
```

Comment:
Nitpick: maybe should be removed?

Reply:
yes to the log (that's left over from me debugging), but the for loops are necessary to download all the files; without them, only the main file is downloaded because only the main file is added to the clipboard.

Reply:
Indeed, I meant the log line

Reply:
Good catch! All fixed.